web/package.json

```diff
@@ -127,7 +127,7 @@
     "@swc-node/register": "~1.6.7",
     "@swc/cli": "~0.1.62",
     "@swc/core": "1.3.96",
-    "@testing-library/react": "14.0.0",
+    "@testing-library/react": "12",
```

Comment:
Is this version change intended to go down?

Reply:
The current unit tests are not compatible with this version